include/uapi/linux/limits.h

```diff
@@ -15,6 +15,7 @@
 #define XATTR_NAME_MAX   255	/* # chars in an extended attribute name */
 #define XATTR_SIZE_MAX 65536	/* size of an extended attribute value (64k) */
 #define XATTR_LIST_MAX 65536	/* size of extended attribute namelist (64k) */
+#define XATTR_LARGE_SIZE_MAX 2097152
```

Comment:
Even though it's obvious, can we add a comment here letting us know what this constant represents?